vars/provisionResources.groovy

```diff
@@ -130,6 +130,10 @@ def call(Map params, String region){
         export SCT_MANAGER_VERSION="${params.manager_version}"
     fi
 
+    if [[ -n "${params.n_db_nodes ? params.n_db_nodes : ''}" ]] ; then
```

Comment:
I don't sure I like this idea of have this as pipeline parameters

seem that I've missed that you introduced it manager pipeline.

is that used in any "formal" triggering ? or just for experimentations ?

Reply:
if it's just for experimentation, https://github.com/scylladb/scylla-cluster-tests/pull/8268 might be a better fit

Reply:
not formal triggers, only manual use

Reply:
I see your solution, will try to add this param in Manager pipeline

Reply:
@fruch 

This change utilizes already existing pipeline parameter, it doesn't add one new to the user interface.
It should be totally ok to do so.

The https://github.com/scylladb/scylla-cluster-tests/pull/8268 looks unrelaed to the goal of this PR.

Reply:
no, I haven't verified longevity pipeline.
So, if there is such probability we will just go with solution suggested by Israel (running the test to verify it right now) and close this one without merging.

Reply:
Yes, it works, closing this PR:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/manager-master/job/ubuntu22-installation-test/7/